rest_specification/get_generate_value.md

```diff
@@ -0,0 +1,22 @@
+# GET: Generate value
+Get generated values for a field 
+
+### Request
+Generate a value using value generators declared for the 'field' of the given entity version. If 'n' is given, generates that many values.
+```
+GET /data/generate/{entityName}/{version}/{field}?n={n}
+```
+Generate a value using value generators declared for the 'field' of the given entities default version. If 'n' is given, generates that many values.
+```
+GET /data/generate/{entityName}/{field}?n={n}
+```
+---
+
```

Comment:
Would be nice to start adding examples.

#### Example

**Request**
```
GET /data/generate/test/path.*.to.field?n=3
```

**Response**
```
[5000,5001,5002]
```